pkg/kn/commands/service/configuration_edit_flags.go

```diff
@@ -264,15 +268,41 @@ func (p *ConfigurationEditFlags) Apply(
 			return fmt.Errorf("only --scale or --scale-max can be specified")
 		} else if cmd.Flags().Changed("scale-min") {
 			return fmt.Errorf("only --scale or --scale-min can be specified")
-		} else {
-			err = servinglib.UpdateMaxScale(template, p.Scale)
+		}
+		if strings.Contains(p.Scale, "..") {
+			scaleParts := strings.Split(p.Scale, "..")
+			if scaleParts[0] != "" {
+				scaleMin, err := strconv.Atoi(scaleParts[0])
+				if err != nil {
+					return err
+				}
+				err = servinglib.UpdateMinScale(template, scaleMin)
+				if err != nil {
+					return err
+				}
+			}
+			if scaleParts[1] != "" {
+				scaleMax, err := strconv.Atoi(scaleParts[1])
+				if err != nil {
+					return err
+				}
+				err = servinglib.UpdateMaxScale(template, scaleMax)
+				if err != nil {
+					return err
+				}
+			}
+		} else if scaleMin, err := strconv.Atoi(p.Scale); err == nil {
+			scaleMax := scaleMin
+			err = servinglib.UpdateMaxScale(template, scaleMax)
 			if err != nil {
 				return err
 			}
-			err = servinglib.UpdateMinScale(template, p.Scale)
+			err = servinglib.UpdateMinScale(template, scaleMin)
 			if err != nil {
 				return err
 			}
+		} else {
+			return fmt.Errorf("Scale must be of the format x..y or x")
```

Comment:
```suggestion
			return fmt.Errorf("scale must be of the format x..y or x")
```

Reply:
error messages should begin with lower case

Reply:
Should I make this edit and create a new PR, or commit suggestion? Not sure how it works after a merge.